src/store/configureStore.dev.js

```diff
@@ -15,11 +15,21 @@ import {General} from 'constants';
 import serviceReducer from 'reducers';
 import deepFreezeAndThrowOnMutation from 'utils/deep_freeze';
 
-import {offlineConfig} from './helpers';
+import {defaultOptions, offlineConfig} from './helpers';
 
-export default function configureServiceStore(preloadedState, appReducer, userOfflineConfig, getAppReducer, enableBuffer = true) {
+export default function configureServiceStore(preloadedState, appReducer, userOfflineConfig, getAppReducer, clientOptions = {}) {
```

Comment:
Can you add a comment here and in the prod file explaining the fields of `clientOptions`? This is arguably the most important function in the library and if the arguments are going to get obscured into an object I think we should document them at least a bit